<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArucoWidget</class>
 <widget class="QWidget" name="ArucoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableDetectionBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable Detection</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="useCalibrationBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Calibration</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="markerSizeBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Marker Size</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
